addons/api/addon/utils/mql-query.js

```diff
@@ -9,8 +9,8 @@
  *
  * @example
  *   const filter = generateMQLFilterExpression({
- *     name: [{ contains: 'search text' }],
- *     scope: [{ equals: 'scope1'}, { equals: 'scope2' }],
+ *     name: { logicalOperator: 'and', values: [{ notEquals: 'search text' }, { notEquals: 'test' }] },
```

Comment:
Non-blocker- can we also include an example when there is no `logicalOperator`?

Reply:
Sure I'll add one